pypeit/core/extract.py

```diff
@@ -1917,8 +1917,6 @@ def ech_objfind(image, ivar, slitmask, slit_left, slit_righ, order_vec, maskslit
         except TypeError:
             std_in = None
 
```

Comment:
Please replace this ToDO. The problem still exists, and it is still thus a TODO.